websites/D/Discord SFX/presence.ts

```diff
@@ -70,6 +73,22 @@ presence.on("UpdateData", async () => {
 		];
 	}
 
+	if (pathname.startsWith("/users/")) {
+		presenceData.details = `Viewing ${
+			document.title.split(" Profile")[0].split(" | ")[1]
+		} profile`;
+
+		try {
+			const { searchParams } = new URL(
+				document.querySelector<HTMLImageElement>('[data-premid-id="avatar"]')
+					?.src || ""
+			);
+			presenceData.largeImageKey = searchParams.get("url");
+		} catch {
+			presenceData.largeImageKey = Assets.Logo;
+		}
+	}
```

Comment:
```suggestion
			const { searchParams } = new URL(
				document.querySelector<HTMLImageElement>('[data-premid-id="avatar"]')
					?.src || ""
			);
			presenceData.largeImageKey = searchParams?.get("url") ?? Assets.Logo;;
	}
```

Reply:
If you don't pass a valid url to the URL, it throws an error. Shouldn't we use try catch?

Reply:
> If you don't pass a valid url to the URL, it throws an error. Shouldn't we use try catch?

My edited suggestion, should work.

Reply:
Lint failed